src/nrc/conf/includes/base.py

```diff
@@ -102,7 +102,7 @@
     "django.contrib.auth",
     "django.contrib.sessions",
     # Note: If enabled, at least one Site object is required
-    # 'django.contrib.sites',
+    "django.contrib.sites",
```

Comment:
do we have the `SITE_ID` envvar/setting configurable?

Reply:
Yes https://github.com/open-zaak/open-notificaties/blob/main/src/nrc/conf/includes/base.py#L24